gradle/dependencies.gradle

```diff
@@ -78,18 +79,18 @@ versions += [
   kafka_21: "2.1.1",
   kafka_22: "2.2.2",
   lz4: "1.5.0",
-  mavenArtifact: "3.6.0",
+  mavenArtifact: "3.8.1",
   metrics: "2.2.0",
   mockito: "2.23.4",
   powermock: "2.0.0",
-  reflections: "0.9.11",
+  reflections: "0.9.12",
   rocksDB: "5.15.10",
   scalatest: "3.0.5",
   scoverage: "1.3.1",
   slf4j: "1.7.25",
   snappy: "1.1.7.2",
   zkclient: "0.11",
-  zookeeper: "3.4.13",
+  zookeeper: "3.4.14",
```

Comment:
How about 3.5.9? I made this change in common.

Reply:
I think it is risky to use 3.5.9 version in 5.2.x. I prefer to go with 3.4.14

Reply:
BTW, https://confluentinc.atlassian.net/browse/KC-1703